lib/elastic_apm/spies/net_http.rb

```diff
@@ -6,6 +6,22 @@ module Spies
     # @api private
     class NetHTTPSpy
       # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
+      class << self
+        def disable_in
+          @disabled = true
+
+          begin
+            yield
+          ensure
+            @disabled = false
+          end
+        end
+
+        def disabled?
+          @disabled ||= false
+        end
+      end
```

Comment:
What if a `NetHTTP` and a `Faraday` request happen at the same time? The `disabled` is set as a class instance variable.

Reply:
You are right. I'll make it thread local.